doc/examples/filters/plot_deconvolution.py

```diff
@@ -2,8 +2,8 @@
 =====================
 Image Deconvolution
 =====================
-In this example, we deconvolve an image using Richardson-Lucy
-deconvolution algorithm ([1]_, [2]_).
+In this example, we deconvolve an image using the Lucy-Richardson /
+Richardson-Lucy algorithm ([1]_, [2]_).
```

Comment:
```suggestion
In this example, we deconvolve an image using the
Richardson–Lucy algorithm ([1]_, [2]_).
```
(it's either "the Richardson–Lucy algorithm" or "Lucy–Richardson deconvolution" according to the Wikipedia article)

Reply:
I think it might be an inconsistency in the wikipedia article, bcoz I did see it being mentioned as "Lucy-Richardson Algorithm" [here](https://www.mathworks.com/help/images/deblurring-images-using-the-lucy-richardson-algorithm.html) and as "Richardson-Lucy deconvolution" [here](https://www.strollswithmydog.com/richardson-lucy-algorithm/). LMK what's best here.

Reply:
The second reference you point to says "the Richardson–Lucy algorithm" and we're citing the Wikipedia article as a reference anyway, so let's stick with "the Richardson–Lucy algorithm" (we do not claim to be exhaustive, and it's not a matter of right/wrong, just self-consistency).